jekyll/_cci1/parallel-manual-setup.md

```diff
@@ -144,7 +156,7 @@ fi
 
 This script partitions the test files into N equally sized buckets, and calls "test-runner" on the bucket for this machine. Note that you will still need to include `parallel: true` in `circle.yml` with this script.
 
-### Test suite split with knapsack gem
+<h3>Test suite split with knapsack gem</h3>
```

Comment:
Actually, coming back around to this--I've seen advice inside and outside CCI discouraging style changes unless it's to make it consistent with the rest of the code.  In this case, I get that you're going for consistent `h2`, `h3` or `##`, `###` usage.

It looks like [other](https://raw.githubusercontent.com/circleci/circleci-docs/master/jekyll/_cci1/build-image-precise.md) [files](https://raw.githubusercontent.com/circleci/circleci-docs/eaad6390837e64fd962bcb92db3c8c34b32eecce/jekyll/_api/v1-reference.md) use the style "prefer hashes for headers unless an HTML id or class are needed".  I would say these `###` to `h3` changes are unnecessary, but your `h2` to `h3` change makes sense.